.github/workflows/test.yml

```diff
@@ -121,10 +121,52 @@ jobs:
       - name: Build Polar WASM library
         run: cargo build -p polar-wasm-api
 
+  build-c-api:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            target
+          key: ${{ runner.os }}-cargo-test-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
I _think_ you want to differentiate the cache keys so both get cached separately.